VRDR.CLI/Program.cs

```diff
@@ -668,6 +668,67 @@ static int Main(string[] args)
                     Console.WriteLine($"{info.Field, -5} {info.Name,-15} {Truncate(info.Contents, 75), -75}: \"{field + "\"",-80}");
                 }
             }
+            else if (args.Length == 2 && args[0] == "extract")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:
+                        var record = submission.DeathRecord;
+                        Console.WriteLine(record.ToJSON());
+                        break;
+                }
+                return 0;
+            }
+            else if (args.Length == 2 && args[0] == "submit")
+            {
+                DeathRecord record = new DeathRecord(File.ReadAllText(args[1]));
+                DeathRecordSubmission message = new DeathRecordSubmission(record);
+                message.MessageSource = "http://mitre.org/vrdr";
+                Console.WriteLine(message.ToJSON());
+                return 0;
+            }
+            else if (args.Length == 2 && args[0] == "resubmit")
+            {
+                DeathRecord record = new DeathRecord(File.ReadAllText(args[1]));
+                DeathRecordUpdate message = new DeathRecordUpdate(record);
+                message.MessageSource = "http://mitre.org/vrdr";
+                Console.WriteLine(message.ToJSON());
+                return 0;
+            }
+            else if (args.Length == 2 && args[0] == "ack")
+            {
+                BaseMessage message = BaseMessage.Parse(File.ReadAllText(args[1]));
+                switch(message)
+                {
+                    case DeathRecordSubmission submission:

```

Comment:
same comment as above re DeathRecordUpdate

Reply:
Removed this switch since it was actually dead code